languages/elixir/exercises/concept/booleans/.docs/introduction.md

```diff
@@ -1,27 +1,27 @@
 Elixir represents true and false values with the boolean type. There are only two values -- _true_ and _false_. These values can be bound to a variable:
 
 ```elixir
-is_true = true
-is_false = false
+true_variable = true
+false_variable = false
 ```
 
 We can combine boolean values into expressions using boolean logical operators.  `and/2`, `or/2`, and `not/1` are commonly used in expressions to evaluate multiple boolean values to a single result.
 
 ```elixir
-is_true = true and true
-is_false = true and false
+true_variable = true and true
+false_variable = true and false
 
-is_true = false or true
-is_false = false or false
+true_variable = false or true
+false_variable = false or false
 
-is_true = not false
-is_false = not true
+true_variable = not false
+false_variable = not true
 ```
 
-In Elixir, when a function returns a boolean value, it often has a `?` for a suffix:
+When writing a function that returns a boolean value, it is idiomatic to end the function name with a `?`.
```

Comment:
I noticed this line is repeated but has a hyperlink in the `after.md` document. Is it intentional to not link here?

Reply:
Yes, often the after.md has more links than the introduction so as to not overly handfeed the answer.